rott/rt_ted.c

```diff
@@ -1334,14 +1425,16 @@ void ReadROTTMap
    long   expanded;
    int    plane;
    byte  *buffer;
+   size_t mapsoffset;
 
    CheckRTLVersion( filename );
+   mapsoffset = GetMapArrayOffset( filename );
```

Comment:
Could you move this one line down and pass the file handle instead of the file name? This would save us an open/close cycle. Likewise for the other two functions calls below.

Reply:
Yes, sure. The same thing could probably be done with `CheckRTLVersion()` in my opinion.

Reply:
> Yes, sure. The same thing could probably be done with `CheckRTLVersion()` in my opinion.

Yes, that's probably a good idea.